src/thunderbots/software/ai/hl/stp/evaluation/find_open_areas.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "ai/world/world.h"
+#include "geom/circle.h"
+
+// TODO: doc comment here
```

Comment:
Not sure where the best place for this evaluation function is, would appreciate suggestions

Reply:
This is fine for now, it's pretty unique so doesn't fit with anything existing.